flask_restful/__init__.py

```diff
@@ -11,7 +11,14 @@
 from flask_restful.utils import http_status_message, unpack, OrderedDict
 from flask_restful.representations.json import output_json
 import sys
-from flask.helpers import _endpoint_from_view_func
+
+try:
+    # Flask < 2.0.0
+    from flask.helpers import _endpoint_from_view_func
+except:
+    # Flask >= 2.0.0
+    from flask.scaffold import _endpoint_from_view_func
```

Comment:
what if we ditch the private method use and copy its implementation (which is just `return view_func.__name__`)?

Reply:
seems like a better solution than importing a private function :)

Reply:
This work for me., Thanks.